Gems/ROS2/Code/Source/RobotImporter/ROS2RobotImporterEditorSystemComponent.h

```diff
@@ -8,15 +8,20 @@
 
 #pragma once
 
+#include "Bus/RobotImporterBus.h"
```

Comment:
Perhaps this header is worth exposing as part of Gem API?

Reply:
@arturkamieniecki

Reply:
Resolved in https://github.com/RobotecAI/o3de-extras/pull/14.